controllers/utils.go

```diff
@@ -1,5 +1,23 @@
 package controllers
 
+import (
+	"fmt"
+
+	gogithub "github.com/google/go-github/v37/github"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// RunnerCache holds cached returns from github runner API calls
+// - default time for cache is 60 * time.Seconds()
+type RunnerCache struct {
+	ExpirationTime metav1.Time
+	Runners        []*gogithub.Runner
+}
+
+func (i *interface{}) test() {
```

Comment:
This is probably not used?

Reply:
The `Runners` is used in the structure to host the actual GitHub Runners from the GitHub client.  I put it in Utils because I was using it in two files - and wasn't sure where else to put it unless I were to make another file specific to this cache inside of the controllers folder

Reply:
I think they were referring to this:

```
func (i *interface{}) test() {
```

Reply:
> I think they were referring to this:
> 
> ```
> func (i *interface{}) test() {
> ```

oh - wow.  Didnt even know that was there.